### DISCUSSION TITLE
[question] Windows 10 x64 install - cannot connect to webpage

### DISCUSSION THREAD

**Discussion by 400HPMustang** [zwave-js/zwave-js-ui#2470] [General]
I ran the installation instructions for "Other Methods" and everything seemed to run but I'm getting the following warnings/messages and I'm assuming that is the reson that zwavejs2mqtt is not working.

How do I work around these so that I can get a working install?

$ yarn install
➤ YN0000: ┌ Resolution step
➤ YN0000: └ Completed in 0s 264ms
➤ YN0000: ┌ Fetch step
➤ YN0000: └ Completed in 0s 553ms
➤ YN0000: ┌ Link step
➤ YN0076: │ fsevents@patch:fsevents@npm%3A2.3.2#~builtin<compat/fsevents>::version=2.3.2&hash=18f3a7 The win32-x64 architecture is incompatible with this module, link skipped.
ncompatible with this module, link skipped.
➤ YN0007: │ core-js@npm:2.6.12 must be built because it never has been before or the last one failed
➤ YN0007: │ docsify@npm:4.12.2 must be built because it never has been before or the last one failed
➤ YN0007: │ core-js@npm:3.21.1 must be built because it never has been before or the last one failed
➤ YN0007: │ leveldown@npm:5.6.0 must be built because it never has been before or the last one failed
➤ YN0007: │ @serialport/bindings-cpp@npm:10.7.0 must be built because it never has been before or the last one failed
➤ YN0007: │ level@npm:5.0.1 must be built because it never has been before or the last one failed
➤ YN0000: └ Completed in 18s 671ms
➤ YN0000: Done with warnings in 19s 740ms

**Comment by robertsLando** [comment]
it's not an error, it's just a warning you can ignore it
